supervision/classification/core.py

```diff
@@ -0,0 +1,226 @@
+            data=image_names,
+            train_ratio=split_ratio,
+            random_state=random_state,
+            shuffle=shuffle,
+        )
+
+        train_dataset = ClassificationDataset(
+            classes=self.classes,
+            images={name: self.images[name] for name in train_names},
+            annotations={name: self.annotations[name] for name in train_names},
+        )
+        test_dataset = ClassificationDataset(
+            classes=self.classes,
+            images={name: self.images[name] for name in test_names},
+            annotations={name: self.annotations[name] for name in test_names},
+        )
+        return train_dataset, test_dataset
+
+    def as_multiclass_folder_structure(
+        self, root_directory_path: str, output_directory_path: str
+    ) -> None:
+        """
+        Saves the dataset as a multi-class folder structure.
+
+        Args:
+            root_directory_path (str): The path to the root directory of the images with which you are working.
+            output_directory_path (str): The path to the directory where the dataset will be saved.
+
+        Example:
+            ```python
+            >>> import supervision as sv
+
+            >>> cd = sv.ClassificationDataset(...)
+
+            >>> cd.as_multiclass_folder_structure(
+            ...     root_directory_path="./images/valid/",
+            ...     output_directory_path="./out",
+            ... )
+        """
+        if not os.path.exists(output_directory_path):
+            os.makedirs(output_directory_path)
+
+        for class_name in self.classes:
+            if not os.path.exists(os.path.join(output_directory_path, class_name)):
+                os.makedirs(os.path.join(output_directory_path, class_name))
+
+            for image in self.annotations:
+                if self.annotations[image].class_id[0] == self.classes.index(
+                    class_name
+                ):
+                    full_dir = os.path.join(root_directory_path, class_name, image)
+
+                    cv2.imwrite(
+                        os.path.join(output_directory_path, class_name, image),
+                        cv2.imread(full_dir),
+                    )
+
+    @classmethod
+    def from_multiclass_folder_structure(
+        cls, root_directory_path: str
+    ) -> "ClassificationDataset":
+        """
+        Load data from a multiclass folder structure into a ClassificationDataset.
+
+        Args:
+            root_directory_path (str): The path to the dataset directory.
+
+        Returns:
+            ClassificationDataset: The dataset.
+
+        Example:
+            ```python
+            >>> import supervision as sv
+
+            >>> cd = sv.ClassificationDataset.from_multiclass_folder_structure(
+            ...     root_directory_path="./dataset",
+            ... )
+        """
+        classes = os.listdir(root_directory_path)
+
+        classes = list(set(classes))
```

Comment:
We can do it simpler. Instead of:

```python
classes = list(set(classes))
classes.sort()
```

You can do:

```python
classes = sorted(set(classes))
```